src/makeRobotAccountant.js

```diff
@@ -7,6 +7,19 @@
 
 function makeRobotAccountant() {
   // write code here
+  let counter = 0;
+
+  return function getSum(a) {
+    return function inner(b) {
```

Comment:
The inner function is unnecessary and makes the code more complex. You can simply add the second parameter to the getSum function.